spot_wrapper/calibration/calibration_util.py

```diff
@@ -203,6 +209,7 @@ def multistereo_calibration_charuco(
     aruco_dict: cv2.aruco_Dictionary = SPOT_DEFAULT_ARUCO_DICT,
     camera_matrices: Optional[Dict] = {},
     dist_coeffs: Optional[Dict] = {},
+    poses: Union[np.ndarray, None] = None,
```

Comment:
`from typing import Optional` and `Optional[np.ndarray]` would probably be easier